keras_retinanet/utils/transform.py

```diff
@@ -205,13 +205,18 @@ def random_transform(
       * flip x (if applied)
       * flip y (if applied)
 
+    Note that by default, the data generators in `keras_retinanet.preprocessing.generators` interpret the translation
+    as factor of the image size. So an X translation of 0.1 would translate the image by 10% of it's width.
+    See `keras_retinanet.utils.image.TransformParameters` for a way to disable this behaviour and use absolute pixel
```

Comment:
Why not directly reference `relative_translation` here?

Reply:
Good point, will change. When I wrote that I hadn't actually implemented it yet, so I didn't know what it was called yet :)

Reply:
Ah I see, well after that it LGTM.